src/bidiMapper/domains/input/ActionDispatcher.ts

```diff
@@ -39,6 +39,10 @@ const CALCULATE_IN_VIEW_CENTER_PT_DECL = ((i: Element) => {
   return [e + ((n - e) >> 1), h + ((m - h) >> 1)];
 }).toString();
 
+const IS_MAC_DECL = (() => {
+  return navigator.platform.toLowerCase().includes('mac');
+}).toString();
```

Comment:
I'd just inline this. If you want to preserve syntax highlighting in text editors, then do:

```
String(() => {
  return ...
})
```

We use this pattern elsewhere

Reply:
Inlining this causes the formatter to modify the declaration's call site drastically. If allowed, I'd prefer to keep it this way consistent with the above decl.

Reply:
OK